Lib/shutil.py

```diff
@@ -813,6 +813,10 @@ def move(src, dst, copy_function=copy2):
             if _destinsrc(src, dst):
                 raise Error("Cannot move a directory '%s' into itself"
                             " '%s'." % (src, dst))
+            if not os.access(src, os.W_OK) and os.listdir(src):
+                raise Error("Cannot move the non-empty directory '%s': "
```

Comment:
f-strings are preferred in new code.

Reply:
Ack, thanks for the flag. But the rest of this file still uses %-formatting for stderr, so I think it's better to maintain consistency. Can we leave f-string conversion for a separate PR?

Reply:
There are conflicting guidelines!

Internal consistency in a module is preferred
Don’t do unrelated changes in a PR
Don’t do cosmetic changes only in a PR

Reply:
Yes, I think I'll prefer internal consistency in this case, and stick with %-formatting.